### DISCUSSION TITLE
Serverpod Tutorial Series

### DISCUSSION THREAD

**Discussion by tylersavery** [serverpod/serverpod#977] [Show and tell]
Hello!

After watching all the content I could find on Serverpod, I put together a tutorial series that covers the workflow in great detail. Would love this communities feedback on this!

https://www.youtube.com/watch?v=oSwT9FZ18j4&list=PLgRx2Eap1Wm1pH9fZz3ISQoCJ-I6wmxFS

**Comment by vlidholt** [comment]
Wow! This is amazing. 🤩 I only had time to look at the first part now, but it looks really good. We'll make sure to help you push this out in social media and add links from our documentation.

**Reply by tylersavery** [reply]
Awesome! I've been following this project for a little while and finally got a chance to test it out myself. Really excited to watch Serverpod grow. :)
